lib/patch_ruby/api/estimates_api.rb

```diff
@@ -35,8 +35,9 @@ def initialize(api_client = ApiClient.default)
     # @param create_bitcoin_estimate_request [CreateBitcoinEstimateRequest] 
     # @param [Hash] opts the optional parameters
     # @return [EstimateResponse]
-    def create_bitcoin_estimate(create_bitcoin_estimate_request, opts = {})
-      data, _status_code, _headers = create_bitcoin_estimate_with_http_info(create_bitcoin_estimate_request, opts)
+    def create_bitcoin_estimate(create_bitcoin_estimate_request = {}, opts = {})
+      _create_bitcoin_estimate_request = Patch::CreateBitcoinEstimateRequest.new(create_bitcoin_estimate_request) 
```

Comment:
I'm sure you tested this locally but would it make sense to add a unit test around this as well to see some of these validations you're describing?

Reply:
Good suggestion! Let me see how we could add that in!